index.html

```diff
@@ -566,6 +566,130 @@ <h1>Trust Model</h1>
 
     <section>
       <h1>Basic Concepts</h1>
+
+      <section>
+        <h2>Types</h2>
+        <p>
+Type information may be expressed via the <a>type</a> property:
+        </p>
+        <dl>
+          <dt><dfn>type</dfn></dt>
+          <dd>
+The value of this property MUST ultimately be one or more URIs.
```

Comment:
The "ultimately" language sounds odd as used here.  I'm guessing this is to say `type` is a URI after JSON-LD processing?  Maybe reword the below "Note" sentence and move it to be second sentence.  Something like "Note that JSON-LD terms that represent URIs MAY be used to ease developer usage."  I'm not sure that's quite right but might make this paragraph less confusing to people not familiar with how JSON-LD works.

Reply:
Agreed, and fixed in https://github.com/w3c/vc-data-model/pull/177/commits/89a757368ab89847c5c61489b68a032c4ca5867e.